waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/FederatedMetaStoreStorage.java

```diff
@@ -31,4 +31,5 @@ public interface FederatedMetaStoreStorage {
 
   void update(AbstractMetaStore oldMetaStore, AbstractMetaStore newMetaStore);
 
+  void saveFederation();
```

Comment:
can you update your branch it looks like the PR contains changes you already merged.

Reply:
Sorry,I will fix it later.

Reply:
> can you update your branch it looks like the PR contains changes you already merged.

Done.